contrib/release/pull-docker-manifests.sh

```diff
@@ -27,39 +31,55 @@ handle_args() {
         common::exit 0
     fi
 
-    if ! echo "$1" | grep -q "[0-9]\+\.[0-9]\+\.[0-9]\+"; then
+    if ! echo "$2" | grep -q "[0-9]\+\.[0-9]\+\.[0-9]\+"; then
+        usage 2>&1
+        common::exit 1 "Invalid VERSION ARG \"$2\"; Expected X.Y.Z"
+    fi
+
+    if [ -z "${GITHUB_TOKEN}" ]; then
         usage 2>&1
-        common::exit 1 "Invalid VERSION ARG \"$1\"; Expected X.Y.Z"
+        common::exit 1 "GITHUB_TOKEN not set!"
     fi
 }
 
+get_digest_output() {
+    local username run_id file tmp_dir archive_download_url archive_download_url_zip
+
+    username="${1}"
```

Comment:
We have other scripts autodetecting the username these days so we can likely remove that parameter, but that's a super minor nit.